modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -58,6 +59,7 @@ public function __construct(
     QueryFactory $entityQuery
   ) {
     $this->node_storage = $entityTypeManager->getStorage('node');
+    $this->term_storage = $entityTypeManager->getStorage('taxonomy_term');
```

Comment:
Needs adding as a protected var further up, even if PHP doesn't seem to care 😆

Reply:
Added the protected member variable for TermStorage